mongomock/filtering.py

```diff
@@ -139,15 +146,15 @@ def _regex(doc_val, regex):
                 '$gte': _not_nothing_and(operator.ge),
                 '$lt': _not_nothing_and(operator.lt),
                 '$lte': _not_nothing_and(operator.le),
-                '$all':_all_op,
-                '$in':lambda dv, sv: any(x in sv for x in _force_list(dv)),
-                '$nin':lambda dv, sv: all(x not in sv for x in _force_list(dv)),
-                '$exists':lambda dv, sv: bool(sv) == (dv is not NOTHING),
+                '$all': _all_op,
+                '$in': lambda dv, sv: any(x in sv for x in _force_list(dv)),
+                '$nin': lambda dv, sv: all(x not in sv for x in _force_list(dv)),
+                '$exists': lambda dv, sv: bool(sv) == (dv is not NOTHING),
                 '$regex': _not_nothing_and(lambda dv, sv: _regex(dv, re.compile(sv))),
                 '$elemMatch': _elem_match_op,
                 }
 
-LOGICAL_OPERATOR_MAP = {'$or':lambda d, subq: any(filter_applies(q, d) for q in subq),
-                        '$and':lambda d, subq: all(filter_applies(q, d) for q in subq),
+LOGICAL_OPERATOR_MAP = {'$or': lambda d, subq: any(filter_applies(q, d) for q in subq),
```

Comment:
I'd probably break the line after the curly braces and decrease the level of indentation for all the following lines, but that is merely a matter of taste.